<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TruthAndCrop</class>
 <widget class="QMainWindow" name="TruthAndCrop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>800</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="settingArea">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>700</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>291</width>
         <height>227</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string> Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="imageField">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string> Output Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="outputPath">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>341</width>
         <height>351</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>310</y>
          <width>291</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="nextBtn">
           <property name="text">
            <string>Next Image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="previousBtn">
           <property name="text">
            <string>Previous Image</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>370</y>
         <width>341</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>311</width>
          <height>191</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Setting(Press your arrow key 'w','a','s','d')</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>StampResolution</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="stampWidthBox">
             <property name="text">
              <string>200</string>
             </property>
             <property name="placeholderText">
              <string extracomment="Width"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="stampHeightBox">
             <property name="text">
              <string>200</string>
             </property>
             <property name="placeholderText">
              <string extracomment="Height"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>arrowKeyIncreaseValue</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="arrowIncreaseBox">
             <property name="text">
              <string>10</string>
             </property>
             <property name="placeholderText">
              <string extracomment="Width"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Prefix Text</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="outputPrefixTextBox">
             <property name="text">
              <string>set_</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>count</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="countBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="settingSaveInfoForObjectDetect">
           <property name="text">
            <string>Save image and info for object detect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="settingNextAfterOneclickBox">
           <property name="text">
            <string>See the next picture after one click.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="inFile">
          <property name="text">
           <string>Input File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="outFile">
          <property name="text">
           <string>Output Path</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>580</y>
         <width>341</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>LogView</string>
       </property>
       <widget class="QTextBrowser" name="logView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>321</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">m</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>854</width>
         <height>838</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="img_view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
